tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -407,7 +415,7 @@ def test_01_multiattach_volumes(self):
             LOG.debug(
                 f"md5sums_before_incr: {md5sums_before_incr}")
 
-            self.snapshot_id2 = self.workload_snapshot(self.wid, False)
+            self.snapshot_id2 = self.workload_snapshot(self.wid, False, snapshot_cleanup=True)
```

Comment:
default cleanup value is True, no need to explicitly pass that parameter again.

Reply:
updated